x-pack/platform/packages/shared/kbn-elastic-assistant/impl/tour/anonymized_values_and_citations_tour/step_config.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiText, EuiSpacer } from '@elastic/eui';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { AI_ASSISTANT_SETTINGS_MENU_CONTAINER_ID } from '../../assistant/assistant_header';
+
+const isMac = navigator.platform.toLowerCase().indexOf('mac') >= 0;
+
+export const anonymizedValuesAndCitationsTourStep1 = {
+  title: (
+    <FormattedMessage
+      id="xpack.elasticAssistant.anonymizedValuesAndCitations.tour.title"
+      defaultMessage="Citations & Anonymized values"
+    />
+  ),
+  subTitle: (
+    <FormattedMessage
+      id="xpack.elasticAssistant.anonymizedValuesAndCitations.tour.subtitle"
+      defaultMessage="New and improved!"
+    />
+  ),
+  anchor: `#${AI_ASSISTANT_SETTINGS_MENU_CONTAINER_ID}`,
+  content: (
+    <EuiText size="s">
+      <FormattedMessage
+        id="xpack.elasticAssistant.anonymizedValuesAndCitations.tour.content.citedKnowledgeBaseEntries"
+        defaultMessage="<bold>Cited</bold> Knowledge base entries show in the chat stream. Toggle on or off in the menu or with the shortcut: {keyboardShortcut}."
+        values={{
+          keyboardShortcut: isMac ? '⌥ c' : 'Alt c',
```

Comment:
```suggestion
          keyboardShortcut: isMac ? '⌥ + c' : 'Alt + c',
```

consider using a `+` in this equation. Looking around the repo and the web for other shortcuts, it appears standard to use the `+`